### ISSUE TITLE
NanoPi M4 - no bluetooth controller found

### ISSUE THREAD

**Issue by darkwaier** [alexreinert/piVCCU#129]
Hi,
I am running [PiVCCU3 Image](https://www.pivccu.de/images/piVCCU3/Armbian_5.68_Nanopim4_Debian_Stretch_dev_piVCCU3_2019-01-08.zip) image on a NanoPi M4 and added iobroker. So far it works fine and the perfomance is great. (ARMBIAN 5.73 stable Debian GNU/Linux 9 (stretch) 4.20.0-rk3399)

To get it fully working again (as I had it in my old setup) I wanted to add my bluetooth sensors, but there seams to be no bluetooth enabled. I went for `armbian-config` and installed bluetooth support. 
Unfortunately this did not work. I did some research and tried various methods to activate it but no luck. There is not default controller available or the kernel does not support it?!

Here are some system printouts...

```
# systemctl status bluetooth
● bluetooth.service - Bluetooth service
   Loaded: loaded (/lib/systemd/system/bluetooth.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2019-02-09 20:29:44 CET; 12min ago
     Docs: man:bluetoothd(8)
 Main PID: 1909 (bluetoothd)
   Status: "Running"
    Tasks: 1 (limit: 4915)
   Memory: 1.1M
      CPU: 25ms
   CGroup: /system.slice/bluetooth.service
           └─1909 /usr/lib/bluetooth/bluetoothd

Feb 09 20:29:44 ioBroker systemd[1]: Starting Bluetooth service...
Feb 09 20:29:44 ioBroker bluetoothd[1909]: Bluetooth daemon 5.43
Feb 09 20:29:44 ioBroker systemd[1]: Started Bluetooth service.
Feb 09 20:29:44 ioBroker bluetoothd[1909]: Starting SDP server
Feb 09 20:29:44 ioBroker bluetoothd[1909]: kernel lacks bnep-protocol support
Feb 09 20:29:44 ioBroker bluetoothd[1909]: System does not support network plugin
Feb 09 20:29:44 ioBroker bluetoothd[1909]: Bluetooth management interface 1.14 initialized
#
```

```
#  hcitool dev
Devices:
#
```

It would be great if somebody with more experience could have a look and give me a hint on how to solve this issue.

Thanks
Thomas

**Comment by alexreinert** [comment]
As I do currently not have a test device, I cannot really help you.
Does Bluetooth work in the nightly image of Armbian?

**Comment by darkwaiter** [comment]
The standard image worked fine but I was not able to get the CCU working (I suppose the bridge did not work properly).
I also did not use the update feature in the armbian-config up to now. I tried it once with the image mentioned in the previous post but after the update the CCU did not work/connect any more. Unfortunately I did not test bluetooth at that time.
If I will have the time (unfortunately not within the next 2 weeks) I will take a second look and will give you an update...
